.travis.yml

```diff
@@ -1,7 +1,18 @@
 language: ruby
+cache: bundler
 rvm:
-  - 2.4.1
-  - 2.3.4
-  - 2.2.7
+  - 2.2.9
+  - 2.3.6
+  - 2.4.3
+  - 2.5.0
+before_install:
+  - gem update --system
+  - gem update bundler
+before_script:
+  - curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./cc-test-reporter
+  - chmod +x ./cc-test-reporter
+  - ./cc-test-reporter before-build
+after_script:
+  - ./cc-test-reporter after-build --exit-code $TRAVIS_TEST_RESULT
```

Comment:
These before and after script blocks install and configure Code Climate coverage reporting and submit coverage reports to Code Climate on successful builds.

If you want to use Code Climate, there are a couple things to be done to get it set up on Code Climate's side and on Travis Ci's side. I documented some of this [on the IndieWeb wiki](https://indieweb.org/Code_Climate) after we merged indieweb/microformats-ruby#77.

Code Climate isn't a _must have_, of course, but it can be quite helpful for surfacing issues in the code.